src/main/kotlin/blackjack/view/Input.kt

```diff
@@ -0,0 +1,17 @@
+package blackjack.view
+
+import kotlin.system.exitProcess
+
+object Input {
+    fun names(): List<String> {
+        println("게임에 참여할 사람의 이름을 입력하세요.(쉼표 기준으로 분리)")
+        return readLine()?.split(",") ?: exitProcess(-1)
+    }
+
+    fun ask(name: String): Boolean {
+        println("${name}는 한장의 카드를 더 받겠습니까?(y/n)")
+        val ask = readLine().toString()
+        if (ask == "y" || ask == "n") ask else null ?: exitProcess(-1)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        if (ask != "y" || ask != "n") exitProcess(-1)
```